src/test/java/org/candlepin/sync/ProductImporterTest.java

```diff
@@ -409,4 +317,170 @@ public void testImportProductMapReturnsEmptyMapWithNoData() throws IOException {
         assertTrue(output.isEmpty());
     }
 
+    @Test
+    public void importShouldSetMetadataExpireTo1() throws Exception {
```

Comment:
Can we add a spec test that checks all content (from all 4 types of product) imported has this set to 1? Unit tests are good, but for business logic like this a unit test can get lost/broken during refactoring very easily

Reply:
I added spec test to master.